generated/nidaqmx/_grpc_interpreter.py

```diff
@@ -2243,58 +2244,62 @@ def read_analog_scalar_f64(self, task, timeout):
             grpc_types.ReadAnalogScalarF64Request(task=task, timeout=timeout))
         return response.value
 
-    def read_binary_i16(self, task, num_samps_per_chan, timeout, fill_mode):
+    def read_binary_i16(
+            self, task, num_samps_per_chan, timeout, fill_mode, read_array):
         response = self._invoke(
             self._client.ReadBinaryI16,
             grpc_types.ReadBinaryI16Request(
                 fill_mode_raw=fill_mode,
-                num_samps_per_chan=num_samps_per_chan, task=task,
-                timeout=timeout))
+                num_samps_per_chan=num_samps_per_chan, read_array=read_array,
+                task=task, timeout=timeout))
         return response.read_array, response.samps_per_chan_read
 
-    def read_binary_i32(self, task, num_samps_per_chan, timeout, fill_mode):
+    def read_binary_i32(
+            self, task, num_samps_per_chan, timeout, fill_mode, read_array):
         response = self._invoke(
             self._client.ReadBinaryI32,
             grpc_types.ReadBinaryI32Request(
                 fill_mode_raw=fill_mode,
-                num_samps_per_chan=num_samps_per_chan, task=task,
-                timeout=timeout))
+                num_samps_per_chan=num_samps_per_chan, read_array=read_array,
+                task=task, timeout=timeout))
         return response.read_array, response.samps_per_chan_read
 
-    def read_binary_u16(self, task, num_samps_per_chan, timeout, fill_mode):
+    def read_binary_u16(
+            self, task, num_samps_per_chan, timeout, fill_mode, read_array):
         response = self._invoke(
             self._client.ReadBinaryU16,
             grpc_types.ReadBinaryU16Request(
                 fill_mode_raw=fill_mode,
-                num_samps_per_chan=num_samps_per_chan, task=task,
-                timeout=timeout))
+                num_samps_per_chan=num_samps_per_chan, read_array=read_array,
+                task=task, timeout=timeout))
         return response.read_array, response.samps_per_chan_read
```

Comment:
This introduces new errors into GrpcStubInterpreter: `read_array` does not exist in the request message.

The right output is something like
```
            grpc_types.ReadBinaryU16Request(
                fill_mode_raw=fill_mode,
                num_samps_per_chan=num_samps_per_chan,
                task=task, timeout=timeout))
        read_array[:] = response.read_array
        return response.samps_per_chan_read
```

@Sakthi-SM Do you intend to address gRPC read/write in a separate "gRPC Interpreter" PR?

Reply:
Yes Brad that will be addressed in a different PR.